test/e2e/storage/persistent_volumes-local.go

```diff
@@ -227,7 +227,7 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 
 				BeforeEach(func() {
 					By("Creating pod1")
-					pod1, pod1Err = createLocalPod(config, testVol)
+					pod1, pod1Err = createLocalPod(config, testVol, framework.PodStartTimeout)
```

Comment:
Let's just make the timeout 1 minute. Can you define a new constant?

Reply:
@msau42 
Add two constants `PodStartShortTimeout` and `ClaimProvisionShortTimeout`, see https://github.com/kubernetes/kubernetes/pull/62635/commits/b08a423a2159c25dca70351cf90fd87059cc02a5.

Should we change all timeouts in `persistent_volumes-local.go` to use `PodStartShortTimeout`?

Reply:
Sure sounds fine

Reply:
Done in https://github.com/kubernetes/kubernetes/pull/62635/commits/47984191a38da76986eb12147d12d718603bdd5e.